src/main/java/core/basesyntax/CreateFigures.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum CreateFigures {
+    CIRCLE,
+    RECTANGLE,
+    RIGHTTRIANGLE,
+    SQUARE,
+    ISISCELESTRAPEZOID
```

Comment:
```suggestion
    RIGHT_TRIANGLE,
    SQUARE,
    ISISCELES_TRAPEZOID
```
Constant names should be separated using underscore